lib/webpacker/dev_server_runner.rb

```diff
@@ -0,0 +1,85 @@
+require "shellwords"
+require "yaml"
+require "socket"
+
+module Webpacker
+  class DevServerRunner
+    def self.run(argv)
+      $stdout.sync = true
+
+      new(argv).run
+    end
+
+    def initialize(argv)
+      @argv = argv
+
+      @app_path          = File.expand_path("../", __dir__)
+      @config_file       = File.join(@app_path, "config/webpacker.yml")
+      @node_modules_path = File.join(@app_path, "node_modules")
+      @webpack_config    = File.join(@app_path, "config/webpack/#{ENV["NODE_ENV"]}.js")
```

Comment:
May be we can load options from `Webpacker.config` or `dev_server` itself instead of loading the yml again?

Reply:
It would be great to avoid that duplication. Currently, `Webpacker.dev_server` inherits form `Webpacker.instance` which depends on `Rails` (for `Rails.root` [here](https://github.com/rails/webpacker/blob/7459f1f6a2a454aa83b68aa609ce88773cd3ee75/lib/webpacker/instance.rb#L6)) and I think it would be preferable to avoid depending on Rails to execute these scripts. Perhaps a future improvement?

Reply:
Makes sense 👍